src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -2510,6 +2510,11 @@ std::pair<std::shared_ptr<Pane>, std::shared_ptr<Pane>> Pane::_Split(SplitDirect
     // Clear out our ID, only leaves should have IDs
     _id = {};
 
+    // Make sure to propagate our _isDefTermSession value
+    WalkTree([&](auto p) {
+        p->_isDefTermSession = _isDefTermSession;

```

Comment:
Wait, I don't know if this is correct... we don't want *all* children of a DefTerm pane to be considered DefTerm, right? Otherwise random new splits will inherit unexpected behavior...

Reply:
I must be misunderstanding something - shouldn't any split made on a def term pane result in everything below it also being def term? I assumed there was no way for a def term session to end up having a pane that is not def term

Reply:
(Resolving this so the convo can continue in the other one)